pkg/cloudprovider/aws/fleet/capacity.go

```diff
@@ -41,6 +42,7 @@ func (c *Capacity) Create(ctx context.Context, constraints *cloudprovider.Constr
 		return nil, fmt.Errorf("computing bin packing, %w", err)
 	}
 
+	zap.S().Debugf("Successfully packed %d pod(s) onto %d node(s)", len(constraints.Pods), len(instancePackings))
```

Comment:
I would say "computed packings of %d pod(s) ...". Packed implies that they were scheduled, imo